src/dackar/utils/mbse/customMBSEparser.py

```diff
@@ -256,4 +269,63 @@ def printOnFiles(self,nodes_file,edges_file):
         self.nodesDf.to_csv(nodes_file, index=False)
         self.edgesDf.to_csv(edges_file, index=False)
 
+    def plot(self, fileID):
+        """
+        Method designed to plot on file the obtained graph
+
+        Args:
+
+            fileID, string, name of the file
 
+        Returns:
+
+            None
+        """  
+        G = nx.DiGraph()
+
+        for index, row in self.edgesDf.iterrows():
+            if row['type'] == 'link':
+                G.add_edge(row['sourceNodeId'],	row['targetNodeId'], type='link')
+            else: 
+                G.add_edge(row['sourceNodeId'],	row['targetNodeId'], type='support')
```

Comment:
There are three types in self.allowedEdgeTypes = ['link','composition','support'], but only two are used here. I may recommend to use elif to check it specifically.